procedures/foreman/branch.md.erb

```diff
@@ -123,6 +123,30 @@ The next step should only be done after all branching, including packaging, has
     - [ ] Remove the oldest version to keep last 3 from the files/directories in the previous steps
 - [ ] Add <%= release %> to [Forklift versions config](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml)
 
+# Prepare Staging Repositories
```

Comment:
This now duplicates the Release process. Traditionally this was not needed because we always made sure we could do the release on the same day and you'd move over to the Release procedure. Now we failed on that. I'd actually like to get back into a rhythm where we release RC1 directly after branching.

Reply:
This has some of the same steps as the release process but it does not duplicate the release process as I see it. There can be cases where it as advantageous to prepare the staging repositories while the sources are being prepped. Case in point from 2.1 was that hammer was being finalized and PRs to the 2.1 branch were failing repoclosure. I could have performed this initial signing and prep work to ensure pipelines were working and that repoclosure would pass for updates to the 2.1 branch.

Reply:
I do think branch -> RC1 same day is a great goal. This is just aimed at doing as much preparation work as possible in parallel. To wit, I think helps the stated goal.